index.js

```diff
@@ -173,9 +211,28 @@ function getOlderCars(/* code here */) {
  * getGermanCars returns an array containing all the cars
  * made by either `Audi` or `Mercedes-Benz` or `Volkswagen` or `BMW`,
  * in the same order as they appear in the original inventory.
-*/
-function getGermanCars(/* code here */) {
-  /* code here */
+ */
+function getGermanCars(inventory) {
+    let germanCars = [];
+
+    for (let i = 0; i < inventory.length; i++) {
+        switch (inventory[i].car_make) {
+            case 'Audi':
+                germanCars.push(inventory[i]);
+                break;
+            case 'Mercedes-Benz':
+                germanCars.push(inventory[i]);
+                break;
+            case 'Volkswagen':
+                germanCars.push(inventory[i]);
+                break;
+            case 'BMW':
+                germanCars.push(inventory[i]);
+                break;
+        }
+    }
+
+    return germanCars
 }
```

Comment:
Good job with this problem.  I like that you used a switch statement instead of an if statement.  It's much more legible than the if statement.